tests/backup/backup_helper.go

```diff
@@ -2283,3 +2284,22 @@ func RemoveElementByValue(arr interface{}, value interface{}) error {
 	}
 	return nil
 }
+
+// RemoveLabelFromNodesIfPresent remove the given label from the given node if present
+func RemoveLabelFromNodesIfPresent(node node.Node, expectedKey string) error {
+	nodeLabels, err := core.Instance().GetLabelsOnNode(node.Name)
+	if err != nil {
+		return err
+	}
+	for key := range nodeLabels {
+		if key == expectedKey {
+			log.InfoD("Removing the applied label with key %s from node %s", expectedKey, node.Name)
+			err = Inst().S.RemoveLabelOnNode(node, expectedKey)
```

Comment:
How about we use Inst().S.RemoveLabelOnNode(node, expectedKey) directly?

Reply:
As discussed, this is added in JustAfterEach in case the remove label in testcase fails.
If we use Inst().S.RemoveLabelOnNode(node, expectedKey) directly and the label is not added,it will throw error

Reply:
Got it. Thanks!